### ISSUE TITLE
Add link to this repo on the website

### ISSUE THREAD

**Issue by Yiyiyimu** [api7/contributor-graph#104]
Maybe we could add a link to the website to guide users contributing (or just star), or people even might not know it's opensource 🤣

**Comment by Baoyuantop** [comment]
Maybe we can add a Github logo link on this page of API7.

**Comment by LiteSun** [comment]
> Maybe we can add a Github logo link on this page of API7(contributor-graph page).

agree, it would be very useful to introduce our repo! what do you think? cc @juzhiyuan
